blocks/math.js

```diff
@@ -32,32 +32,24 @@ goog.require('Blockly.Colours');
 
 goog.require('Blockly.constants');
 
-
-/**
- * Common HSV hue for all blocks in this category.
- */
-Blockly.Blocks.math.HUE = Blockly.Colours.textField;
-
 Blockly.Blocks['math_number'] = {
   /**
    * Block for generic numeric value.
    * @this Blockly.Block
    */
   init: function() {
```

Comment:
Fair enough. My intention in keeping the min blocks/ is that they could be shared between the horizontal and vertical (as they currently are). Maybe we should get rid of blocks/ and put our shared blocks in a blocks_common/ to avoid merge conflicts?